screens/UsingScreen.js

```diff
@@ -21,6 +21,27 @@ const UsingScreen = props => {
   const { timeRemaining } = time;
 
   const startAlarm = () => {
+    // Check if location has been set
+    if (
+      location === null ||
+      location.coords === null ||
+      (location.coords.lat === 0 && location.coords.lng === 0)
+    ) {
```

Comment:
The LoadingScreen actually fetches the lat-lon coordinates when its loading in. So it will never be null/0 if it fetches gps location successfully.

Reply:
I found a case that if a device's location is deactivated somehow, the fetch will fail - and by default redux initializes the location states as:
```
location: {
    coords: {
        lat: 0, lng: 0
    }
}
```